templates/dashboard/base.html

```diff
@@ -86,67 +86,97 @@
                   <a href="{% url 'dashboard:category-list' %}">
                     {% trans "Categories" context "Dashboard categories list" %}
                   </a>
                 </li>
+                {% endif %}
+                {% if perms.product.view_stock_location %}
                 <li class="{% block menu_location_class %}{% endblock %}">
                   <a href="{% url 'dashboard:product-stock-location-list' %}">
                   {% trans "Stock locations" context "Dashboard stock loactions list" %}
                   </a>
                 </li>
+                {% endif %}
               </ul>
             </li>
+            {% endif %}
+            {% if perms.discount.view_sale or perms.discount.view_voucher %}
             <li class="side-nav-section">
               <p>
                 {% trans "Discounts" context "Dashboard discounts list" %}
               </p>
               <ul>
+                {% if perms.discount.view_sale %}
                 <li class="{% block menu_sales_class %}{% endblock %}">
                   <a href="{% url 'dashboard:sale-list' %}">
                     {% trans "Sales" context "Dashboard sales (discounts) list" %}
                   </a>
                 </li>
+                {% endif %}
+                {% if perms.discount.view_voucher %}
                 <li class="{% block menu_vouchers_class %}{% endblock %}">
                   <a href="{% url 'dashboard:voucher-list' %}">
                     {% trans "Vouchers" context "Dashboard vouchers list" %}
                   </a>
                 </li>
+                {% endif %}
               </ul>
             </li>
+            {% endif %}
+            {% if perms.order.view_order or perms.userprofile.view_user %}
             <li class="side-nav-section">
               <p>
                 {% trans "Sales" context "Dashboard sales. `Sales' is noun of to sell" %}
               </p>
               <ul>
+                {% if perms.order.view_order %}
                 <li class="{% block menu_orders_class %}{% endblock %}">
                   <a href="{% url 'dashboard:orders' %}">
                     {% trans "Orders" context "Dashboard orders list" %}
                   </a>
                 </li>
+                {% endif %}
+                {% if perms.userprofile.view_user %}
                 <li class="{% block menu_customers_class %}{% endblock %}">
                   <a href="{% url 'dashboard:customers' %}">
                     {% trans "Customers" context "Dashboard customers list" %}
                   </a>
                 </li>
+                {% endif %}
               </ul>
             </li>
+            {% endif %}
             <li class="side-nav-section">
               <p>
                 {% trans "Configuration"  context "Dashboard configuration" %}
               </p>
               <ul>
+                <li class="{% block menu_staff_class %}{% endblock %}">
```

Comment:
"Staff members" and "Groups" sections should probably be visible only to superusers.

Reply:
Yea i have a question if we need some additional User model permissions for staff like the whole Configuration in Dashboard or do we just use superuser.

Reply:
Decided to cover every view and template behind superuser check.